### ISSUE TITLE
Doesn't work with currency formatted numbers

### ISSUE THREAD

**Issue by nikhil-thadani** [heyman333/react-animated-numbers#46]
It's good if we use it just for numbers, but if we need to use it with real world data i.e. formatted numbers according to currency then it's of no use, cause the formatted string can't be accepted in `animateToNumber` prop.

**Comment by doliG** [comment]
Agree, it could be nice to add a prepend / append option to add locale
